doc/content/getting-started/ttn/_index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "The Things Network"
+description: ""
+distributions: "The Things Network"
+---
+
+{{< figure src="logo_TTN.svg" class="float plain" width="200px" >}}
+
+The Things Network is a global community building an open, crowdsourced, decentralized LoRaWAN network.
```

Comment:
```suggestion
The Things Network is a global collaborative Internet of Things ecosystem that creates networks, devices and solutions using LoRaWAN.
```

Reply:
It's important to frame TTN not only as the network, but rather the ecosystem, which also provides and maintains a public network.

Reply:
Fine with me, but the original sentence has been the initiative description since 2015. I agree it has evolved over time. Let's not block this PR on this though.

Reply:
This sentence is the updated one which we will also use on the new TTN website. cc @wienke @laurensslats